Bikeini/utilities/deviceStorage.js

```diff
@@ -16,16 +18,17 @@ const deviceStorage = {
 	    try {
 	      const value = await AsyncStorage.getItem('id_token');
 	      if (value !== null) {
-	      	console.log(value);
+        console.log('found:', value);
```

Comment:
Is this one neccessary?

Reply:
nahh but it might be removed later, must verify its not in use anymore first